Timer/force-app/main/default/lwc/timerLWC/timerLWC.js

```diff
@@ -0,0 +1,97 @@
+import { LightningElement } from "lwc";
+
+export default class TimerLWC extends LightningElement {
+  showStartBtn = true;
+  showHoursMinutesSeconds = true;
+  showSeconds = false;
+  showMinutes = false;
+  showHours = false;
+  seconds = 0;
+  minutes = 0;
+  hours = 0;
+
+  handleChange(event) {
+    const field = event.target.name;
+    if (field === "seconds") {
+      this.seconds = event.target.value;
+      if (this.seconds > 0) {
+        this.showSeconds = true;
+      }
+    } else if (field === "minutes") {
+      this.minutes = event.target.value;
+      if (this.minutes > 0) {
+        this.showMinutes = true;
+      }
+    } else if (field === "hours") {
+      this.hours = event.target.value;
+      if (this.hours > 0) {
+        this.showHours = true;
+      }
+    }
+  }
+
+  timeIntervalInstance;
+
+  start(event) {
+    this.showStartBtn = false;
+    this.showHoursMinutesSeconds = false;
+    const parentThis = this;
+
+    this.timeIntervalInstance = setInterval(function () {
+      if (parentThis.seconds <= 0 && parentThis.minutes > 0) {
```

Comment:
Too much complexity in a single component. If you create that separate component which is mentioned in first comment, most of the logic will be encapsulated in it.